src/fparser/one/parsefortran.py

```diff
@@ -130,16 +129,16 @@ def parse(self):
             raise
         except Exception as error:
             reader = self.reader
+            logger = logging.getLogger(__name__)
             while reader is not None:
                 message = \
                     reader.format_message('FATAL ERROR',
                                           'while processing line',
                                           reader.linecount, reader.linecount)
-                logging.getLogger(__name__).critical(message)
+                logger.critical(message)
```

Comment:
Bug fix?

Reply:
Just a tidy up. It avoids making the `logging.getLogger(__name__)` call multiple times in the same function.